documentation/Experian Commerce Cloud Integration Manual.md

```diff
@@ -342,14 +336,15 @@ Once the EDQ SitePreferences are uploaded in the business manager; choose your s
    1. Global Intuitive
    1. Verification Engine
    1. None
-7.	Address layout (String): Address layout format for the data.
-8. Staging (Enum of String): Select one option between:
-   1. Production
-   1. Development
-9. Data Set Code (String): Code to use Data Sets in case the country ISO code requires it.
-10. Activates or deactivates Data Set usage (Boolean): Activates or deactivates Data Set usage (only use for Non USA).
-11. Pro Web Custom Transition Callback (String): Name for the custom customer function to be triggered.
-12. Pro Web Callback validation (Boolean): Activates or deactivates page transition from the callback.
+7. Address layout (String): Address layout format for the data. Examples:
+   1. EDQ NA Integration 5 Line No Retention - North America Integration Layout Without Any Retention Elements
+   1. EDQ NA Integration 5 Line Default - North America Integration Layout With Sub Premise And Care Of Retention
+   1. EDQ NA Integration 5 Line Modified - North America Integration Layout With Sub Premise And Care Of Retention But With All Secondary Information Split Out For Concatenation
+   1. EDQ NA Integration 5 Line Full Retention - North America Integration Layout With Sub Premise, Care Of, Pre Street, and Post Street Retention
```

Comment:
Let's leave this out for now as these layouts are not actually in production quite yet. Once they're pushed to production we can update this. Also, you'll either want to number them numerically (e.g. 1, 2, 3) or just use bullet points. Right now what you have is kind of confusing. 

I also notice it's like that for the other ones (e.g. _Email Enable_) so let's change that for all of them.